docs/website/blog/2023-11-27-dlt-data-lineage.md

```diff
@@ -0,0 +1,259 @@
+
+		load_a = pipeline_store.run_pipeline(data=select_c_data_store, table_name='sales_info', write_disposition='replace')
+    load_b = pipeline_shopify.run_pipeline(data=select_c_data_shopify, table_name='sales_info', write_disposition='replace')
+
+    pipeline_store.run_pipeline(data=load_a.load_packages, table_name="load_info", write_disposition="append")
+    pipeline_shopify.run_pipeline(data=load_b.load_packages, table_name='load_info', write_disposition="append")
+```
+
+### Step 3: Run the dlt pipeline.
+
+To run the pipeline execute the following command:
+
+```python
+python data_lineage.py
+```
+
+This will load the data into BigQuery. We now need to remove the column filters from the code and rerun the pipeline. This will add the filtered columns to the tables in BigQuery. The change will be captured by `dlt`.
+
+## Data Transformation and Lineage
+
+Now that both the Shopify and Store data is available in BigQuery we will use **dbt** to transform the data.
+
+### Step 4: Initialize a dbt project and define model
+
+To get started initialize a dbt project in the root directory:
+
+```python
+dbt init sales_dbt
+```
+
+Next, in the `sales_dbt/models` we define the dbt models. The first model will be the `fact_sales.sql`. The skate shop has two data sources the online Shopify source and the physical Store source. We need to combine the data from both sources to create a unified reporting feed. The **fact_sales** table will be our unified source.
+
+Code for `fact_sales.sql`:
+
+```sql
+{{ config(materialized='table') }}
+
+select 
+  invoice_id,
+  city,
+  unit_price,
+  quantity,
+  total,
+  date,
+  payment,
+  info._dlt_id,
+  info._dlt_load_id,
+  loads.schema_name,
+  loads.inserted_at
+from {{source('store', 'sales_info')}} as info
+left join {{source('store', '_dlt_loads')}} as loads 
+on  info._dlt_load_id = loads.load_id
+
+union all
+
+select 
+  name as invoice_id,
+  billing_city,
+  lineitem_price,
+  lineitem_quantity,
+  total,
+  created_at,
+  payment_method,
+  info._dlt_id,
+  info._dlt_load_id,
+  loads.schema_name,
+  loads.inserted_at
+from {{source('shopify', 'sales_info')}} as info
+left join {{source('shopify', '_dlt_loads')}} as loads
+on  info._dlt_load_id = loads.load_id
+where financial_status = 'paid'
+```
+
+In the query, we join the sales information for each source with its dlt load_info. This will help us keep track of the number of rows added with each pipeline run. The `schema_name` identifies the source that populated the table and helps establish the table lineage. While the `_dlt_load_id` identifies the pipeline run that populated the each row and helps establish row level lineage. The sources are combined to create a **fact_sales** table by doing a union over both sources. 
+
+Next, we define the **`schema_change.sql`** model to capture the changes in the table schema using following query:
```

Comment:
why the model name is bold: **`schema_change.sql`**
in this case, the names of all models must be in bold or not all in bold.